site/layouts/partials/footer.html

```diff
@@ -10,7 +10,7 @@ <h3 class="accordion-header footer-heading" id="headingLinks">
             </button>
             <span class="d-none d-md-flex">Links</span>
           </h3>
-          <div id="collapseLinks" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
+          <div id="collapseLinks" class="container-xxl accordion-collapse collapse" aria-labelledby="headingLinks" data-bs-parent="#sitemapAccordion">
```

Comment:
You forgot to remove those modifications linked to the accordion Bootstrap PR no?

Reply:
I think that those one are just changes for consistency regarding the way accordion are built ! May I revert this as well ?